packages/quantic/force-app/main/default/lwc/quanticCategoryFacetValue/quanticCategoryFacetValue.html

```diff
@@ -2,15 +2,15 @@
   <li class={categoryFacetLiClass}>  
     <div class="slds-size_1-of-1" onclick={onSelect}>
       <template if:true={nonActiveParent}>
-        <div class="facet__value-option slds-grid slds-grid_vertical-align-center slds-size_1-of-1 slds-p-around_x-small">
+        <div class="facet__parent-value-option slds-grid slds-grid_vertical-align-center slds-size_1-of-1 slds-p-around_x-small">
```

Comment:
this change affects how the css is applied. I suggest keeping the previous class and adding an extra one.